server/api/users.js

```diff
@@ -1,16 +1,8 @@
 const router = require('express').Router()
 const {User} = require('../db/models')
+const {isAdmin} = require('./util')
 module.exports = router
 
-const isAdmin = (req, res, next) => {
```

Comment:
I used isAdmin in another file (products.js) so I thought it will be better to move isAdmin to separate file (util.js) and export since it's used in more files.